lib/Config.ts

```diff
@@ -122,6 +122,7 @@ class Config {
       disable: false,
       host: 'localhost',
       port: 5001,
+      cltvdelta: 5760,
```

Comment:
Value should be 5760 to allow second leg of direct BTC .

BTC final CLTV - 144 blocks -> 1440 minutes ->  5760 eth blocks  
Need to add 100 blocks for the raiden leg itself.

Reply:
This calculation should be done at run time. 
The configuration value should be 100  Eth blocks. This is what raiden is expecting the lock to be. We need to add to 100 the length of the first-leg.

Reply:
Related to the comment above (as this is the same config parameter), this number gets added to the total delay of the 2nd leg when calculating the `makerCltvDelta` for the delay of the final hop of the first leg of the payment.

Reply:
@erkarl If we make it global (as you mentioned above) and calculate it based on the block times, then we wouldn't be able to have a raiden specific value here. What do you have in mind?